google-cloud-storage/src/main/java/com/google/cloud/storage/StorageOptions.java

```diff
@@ -110,6 +111,9 @@ public abstract static class Builder
     public abstract StorageOptions.Builder setBlobWriteSessionConfig(
         @NonNull BlobWriteSessionConfig blobWriteSessionConfig);
 
+    public abstract StorageOptions.Builder setOpenTelemetrySdk(
```

Comment:
`@BetaApi` and updated javadoc comment noting otel integration is in preview and subject to change without notice.